installing/installing_bare_metal/installing-bare-metal.adoc

```diff
@@ -77,11 +77,9 @@ include::modules/installation-approve-csrs.adoc[leveloffset=+1]
 
 include::modules/installation-operators-config.adoc[leveloffset=+1]
 
-include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
-
-include::modules/registry-configuring-storage-baremetal.adoc[leveloffset=+3]
+include::modules/registry-removed.adoc[leveloffset=+2]
 
-include::modules/installation-registry-storage-non-production.adoc[leveloffset=+3]
+include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
```

Comment:
This module should either only exist in the registry/configuring-registry-storage/configuring-registry-storage-baremetal.adoc assembly or be removed entirely. Is that correct?